README.md

```diff
@@ -1 +1,190 @@
-# inpycon2021-tasks
\ No newline at end of file
+# PyCon India 2021
+
+
+## Event Dates:
+- Conference: 16th and 17th Sep, 2021
+- Workshop: 18th Sep, 2021
+- DevSprint: 18th and 19th Sep, 2021
+
+## Conference Structure
+- Platform: Hopin
+- Communication: Zulip Chat
+- 5 tracks on stage
+    * 3 general tracks
+    * Diversity
```

Comment:
Also it would be better to explain the terminology:
- What is Tracks?
- What is Sessions?